examples/resources/iam_policy/resource_iam_cluster.tf

```diff
@@ -0,0 +1,29 @@
+# Create Role Bindings for Cluster Scope
+resource "tanzu-mission-control_iam_policy" "create_iam_policy_cluster" {
```

Comment:
"create_iam_policy_cluster" to "cluster_scoped_iam_policy".  create is not the right terminology because the same resource is used for create, update , delete .